mappings/net/minecraft/client/gui/InteractionElement.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS cws net/minecraft/client/gui/ScreenChild
+CLASS cws net/minecraft/client/gui/InteractionElement
```

Comment:
`InteractableElement`?

Reply:
ok that works

Reply:
or it could just be an `Element` because there's no non-interactable elements?

Reply:
Sounds good 👍